CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# dbt_netsuite v0.6.1
+
+PR [#42](https://github.com/fivetran/dbt_netsuite/pull/42) includes the following update to the dbt_netsuite package:
```

Comment:
This is really only needed if we have a large number of updates in a single PR. Since this is just one PR, we can identify the PR in the specific change below. Can we remove this line?

Reply:
removed